### PR TITLE
feat(snyk_scan): add snyk.io docker image scan

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

**Comment by revolunet** [comment]
we could add it to autodevops in a 2nd step

File: base_snyk_scan.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
+# Base job to run Snyk scan on docker image
+#
+# Run a scan on the main image using `SNYK_TOKEN` CI variable
+#
+
+.base_snyk_scan:
+  image: snyk/snyk-cli:docker
```
Comment (douglasduteil) [inline base_snyk_scan.yml:8]:
```suggestion
  image: snyk/snyk-cli:1.305.0-docker
```

File: base_snyk_scan.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  services:
+    - docker:$DOCKER_VERSION-dind
+  dependencies: []
+  when: manual
+  allow_failure: true
+  variables:
+    IMAGE: "${CI_REGISTRY_IMAGE}:${CI_COMMIT_SHA}"
+  script:
+    - echo "$CI_JOB_TOKEN" | docker login $CI_REGISTRY -u gitlab-ci-token --password-stdin
+    - echo $IMAGE
```
Comment (douglasduteil) [inline base_snyk_scan.yml:18]: Is it required ?

Comment (revolunet) [reply base_snyk_scan.yml:18]: nope, debug

**Comment by revolunet** [comment]
fixed

**Pull request merged**